python/pyspark/ml/feature.py

```diff
@@ -167,6 +168,137 @@ def getSplits(self):
 
 
 @inherit_doc
+class CountVectorizer(JavaEstimator, HasInputCol, HasOutputCol):
+    """
+    Extracts a vocabulary from document collections and generates a [[CountVectorizerModel]].
```

Comment:
* Use sphinx link instead of `[[...]]`. You can check the generate HTML doc and see whether the links are broken.
* docstring line width is 72 defined by PEP8.